src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -266,6 +266,17 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Formats {@code dateInUtc} according to the ISO8601 format.
+     */
+    public static String formatDateToIso8601Utc(Date dateInUtc) {
+        if (dateInUtc == null) {
+            return "";
+        }
+        SimpleDateFormat sdf = new SimpleDateFormat(Const.TIME_FORMAT_ISO_8601_UTC);
+        return sdf.format(dateInUtc);
```

Comment:
Travis runs on UTC. This system (#7521 and the setting of time zone for every `SimpleDateFormat` instance) is in place for ease of local testing on systems that don't run UTC, and has no effect on systems that are already on UTC.